ecosystem/typescript/sdk_v2/src/bcs/serializer.ts

```diff
@@ -235,9 +250,9 @@ export class Serializer {
    *
    * @example
    * // Define the MoveStruct class that implements the Serializable interface
-   * class MoveStruct implements Serializable {
+   * class MoveStruct extends Serializable {
    *     constructor(
-   *         public creatorAddress: AccountAddress, // where AccountAddress implements Serializable
+   *         public creatorAddress: AccountAddress, // where AccountAddress extends Serializable
```

Comment:
maybe also include `extends Serializable` for our [AccountAddress](https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk_v2/src/core/account_address.ts#L37) class as part of this PR?

Reply:
Ah yes, done!